### ISSUE TITLE
프로젝트 import시 프로젝트 홈 문제

### ISSUE THREAD

**Issue by RetroTV** [yona-projects/yona-export#5]
기존의 프로젝트를 export한 후 새로운 서버에 import시 이슈/마일스톤/게시판은 모두 정상적으로 옮겨졌으나, 홈 쪽의 README/최근 이력/대시보드는 모두 정상적으로 동작하지 않습니다.

YONA 버전은 1.12.0을 사용하고 있습니다.

![README mdX](https://user-images.githubusercontent.com/38521736/56781998-317c9080-6820-11e9-8d6d-ce912bd8ccca.png)
![정상동작X](https://user-images.githubusercontent.com/38521736/56781999-317c9080-6820-11e9-932e-aea75e1f2ad2.png)

**Comment by doortts** [comment]
- README는 표시되지 않고
- 최근이력/대시보드는 클릭시 오류 발생

인 상태인거죠?

**Comment by RetroTV** [comment]
예, 혹시나 권한문제일까 싶어서 테스트용 아이디를 만들어 시도해 보기도하고 어드민 권한을 줘서도 import 해봤습니다만 마찬가지 였습니다.

**Comment by doortts** [comment]
해당 프로젝트 게시글에서 아무글이나 하나 수정 누르셔서 README로 지정 저장해 보시겠어요?

![image](https://user-images.githubusercontent.com/520523/56784460-8c67b500-682b-11e9-9942-0683517ecb6a.png)

**Comment by RetroTV** [comment]
![README](https://user-images.githubusercontent.com/38521736/56784538-e6687a80-682b-11e9-8c83-fcd70137fcea.png)

README.md는 정상적으로 뜨네요, 하지만 최근 이력/대시보드는 여전히 서버 오류가 뜹니다.

**Comment by doortts** [comment]
혹시 서버 로그 첨부 가능하세요?

**Comment by RetroTV** [comment]
yona-export 쓸 때 나오는 로그 인가요, 아니면 yona가 돌아가고 있는 서버쪽 인가요?

**Comment by RetroTV** [comment]
[logs.zip](https://github.com/yona-projects/yona-export/files/3119791/logs.zip)
Import를 시도한 쪽의 로그파일들 입니다.

**Comment by doortts** [comment]
현제 운영중인 서버, 클릭시 오류가 발생하는 서버의 오류 메시지 부탁드릴게요.

**Comment by RetroTV** [comment]
import 시도시 TO쪽 서버에 뜨는 로그
[import_log.txt](https://github.com/yona-projects/yona-export/files/3119837/import_log.txt)

import 완료 후 최근 이력/대시보드 클릭시 뜨는 로그
[import_log2.txt](https://github.com/yona-projects/yona-export/files/3119838/import_log2.txt)

export 쪽에는 별다른 오류 로그가 뜨지 않았으며, 본작업 이전에 Test용으로 생성한 프로젝트 입니다.

**Comment by doortts** [comment]
Fix 를 위해 전용 패치를 하나 만들어서 드리려고 하는데요 H2, MariaDB, 둘 중 어느 버전을 사용하고 계세요?

**Comment by RetroTV** [comment]
MariaDB 10.2 쓰고 있습니다.

**Comment by doortts** [comment]
전용 패치 배포판을 만들어 보았습니다

https://send.firefox.com/download/89650bc3fad204ff/#yU94zOVYhS3zSmh7EKczbw

로 테스트 해 보시겠어요?

**Comment by RetroTV** [comment]
import 시도시 TO쪽 서버에 뜨는 로그
[import_log_patch.txt](https://github.com/yona-projects/yona-export/files/3120281/import_log_patch.txt)

import 완료 후 최근 이력/대시보드 클릭시 뜨는 로그
[import_log2_patch.txt](https://github.com/yona-projects/yona-export/files/3120282/import_log2_patch.txt)

아직 문제가 있는것 같습니다.

제가 잘못 세팅한 걸수도 있으니, 주말동안에 집에서도 테스트 해봐야 할 것 같네요.

**Comment by doortts** [comment]
다시 테스트 하실때는 아래 링크 버전으로 부탁드려요
https://send.firefox.com/download/4cf2cb8448796495/#yTKRcI6lZuPoAPSktC0UzQ

그리고 import 된 쪽 서버 로그 부탁드릴게요.

**Comment by RetroTV** [comment]
예, 알겠습니다.
다음 보고는 좀 늦을거 같으니, 오늘은 기다리지 않으시는게 좋을 것 같습니다!

**Comment by RetroTV** [comment]
현재, 집안에서 완전히 새로 구축한 서버에선 정상적으로 동작하고 있습니다.
월요일에, 진행중인 프로젝트에도 정상적으로 동작하는지 확인 후 다시 답변 드리겠습니다

**Comment by RetroTV** [comment]
일단 테스트용 프로젝트를 import 했을 때, 오류가 나던 페이지는 정상으로 뜨고 있고, 기존의 프로젝트도 정상적으로 import 완료 후 제대로 동작하고 있습니다.
로그는 ParseException 뜨길래 혹시나해서 첨부 했습니다.

[project_import_log.txt](https://github.com/yona-projects/yona-export/files/3126313/project_import_log.txt)

**Comment by doortts** [comment]
로그 고맙습니다. 어느부분에서 문제가 생겼는지 조금더 구체적인 유추가 가능해졌습니다.
프로젝트를 export 하는 쪽의 Yona 버전과 DB가 어떻게 되나요?

**Comment by RetroTV** [comment]
YONA 버전은 1.9.1로 쓰던걸로 기억하는데, MariaDB는 어떤버전을 쓰는지 모르겠습니다.
관리자에게 직접 물어봐야 할 것 같은데, 다음주까지 출근을 안하셔서;;

**Comment by RetroTV** [comment]
답변이 많이 늦어서 죄송합니다.
기존의 YONA 서버는 마리아DB 10.3.10/YONA 1.12.0 버전을 쓰고 있었다고 합니다.

**Comment by doortts** [comment]
예상했던 부분과는 조금 다르네요. 우선 원인은 이슈의 작성일자 부분에 대한 export  Date 포맷을 변경한적이 있는데 그 버전에서 export 했다면 해당 문제가 발생할 수 있다고 생각했었습니다.
그런데 export 를 실행한 쪽의 Yona가 1.12면 최근버전이라 또 그런 문제도 아니네요

**Comment by RetroTV** [comment]
원래 1.9인가 1.9.1 버전대를 사용하시다 업데이트를 했다고 하셨습니다.
MariaDB도 그렇고.

**Comment by RetroTV** [comment]
이전에 주신, 테스트용 yona 버전은 동작하는것 같으니 일단 이걸 사용하겠습니다.
